macros/generic_tests/test_equal_rowcount.sql

```diff
@@ -5,6 +5,6 @@
 select
     abs(count_a - count_b) as diff_count
 from (select count(*) as count_a from {{ model }}) as model_a
-cross join (select count(*) as count_b from "{{ model.schema }}"."{{ compare_model }}") as model_b
+cross join (select count(*) as count_b from {{ model.schema }}.{{ compare_model }}) as model_b
```

Comment:
This does not work in Snowflake, when omitting the quotes it will use the uppercase value of the schema and model